cmd/cmd.go

```diff
@@ -623,32 +625,144 @@ func ShowHandler(cmd *cobra.Command, args []string) error {
 		showType = "template"
 	}
 
-	if flagsSet > 1 {
+	switch flagsSet {
```

Comment:
instead of a `switch` here, we can early return:

```
if len(flagsSet) > 1 {
  return errors.New("only one of '--license', '--modelfile', '--parameters', '--system', or '--template' can be specified")
}

if len(flagsSet) == 1 {
  ... do single flag stuff
  return nil
}

.. do default stuff
return nil